maintenance-filter-controller/src/main/java/org/ccci/maintenance/MaintenanceControlClient.java

```diff
@@ -86,9 +91,54 @@ public Failure(String message, IOException exception)
             }
             log.info("successfully updated window on " + server);
         }
+        if (failures.isEmpty())
+        {
+            printConfirmationOfWindowBoundaries(window);
```

Comment:
When is this logged? When you send client configuration to the maintenance server?